src/server.js

```diff
@@ -35,10 +37,22 @@ export default function start(
 
 	const baseRoutes = getRoutes(renderRequestMap);
 	const finalRoutes = [...routes, ...baseRoutes];
+	let tls = null;
```

Comment:
nitpick, but in general I'd aim to just 'inject' the `connection.tls` property directly inside an `if` block after `connection` is first defined in order to avoid having a re-assignable `let` declaration and to avoid any ambiguity around passing in a `tls` key/val that may or may not be defined.